sdk/python/cap/worker.py

```
@@ -12,133 +13,287 @@
+        return True
+    public_key = public_keys.get(packet.sender_id)
+    if not public_key:
+        logger.warning("no public key found", extra={"sender_id": packet.sender_id})
+        return False
+    signature = packet.signature
+    packet.ClearField("signature")
+    unsigned_data = packet.SerializeToString(deterministic=True)
+    packet.signature = signature
+    try:
+        public_key.verify(signature, unsigned_data, ec.ECDSA(hashes.SHA256()))
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)